tools/msys2_fetch_and_build_all_tx16s.sh

```diff
@@ -1,12 +1,38 @@
 #! /usr/bin/env bash
 
-## Bash script to show how to get EdgeTX source from GitHub (main branch),
+## Bash script to show how to get EdgeTX source from GitHub,
 ## how to build TX16S firmware, Companion, Simulator, radio simulator
 ## library and how to create an installation package.
 ## Let it run as normal user in MSYS2 MinGW 64-bit console (blue icon).
 
-PAUSEAFTEREACHLINE="false"
-STEP=1
+# -----------------------------------------------------------------------------
+export BRANCH_NAME="main"  # main|2.8|2.9|...
+export RADIO_TYPE="TX16S"  # TX16S|X10|X10EXPRESS|X12S|XLITE|XLITES|X9LITE|X9LITES|X9D|X9D+|X9D+2019|X9E|X7|X7ACCESS|T16|T18||T12|TX12|COMMANDO8
```

Comment:
This needs further thought and refinement, as radio target is selected for some targets via a single PCB parameter, whereas some radio targets need two parameters specified - PCB and PCBREV.

Reply:
Also, have a look at line 92 of this very script...
`-DPCB=X10 -DPCBREV=${RADIO_TYPE}`
... X10 is not valid for any of the B&W radios listed - only colorlcd radios, and not even all of them at that (i.e. not the NV14/EL18/PL18). 

Basically, you'll want to have those two parameters, and indicate that one essentially selects the radio family (PCB), and the second specifies the variant (PCBREV). 

This should give you a good idea as to the combinations:
https://github.com/EdgeTX/edgetx/blob/main/tools/build-gh.sh#L95-L178

Reply:
Ok
Will change that